sdk/attestation/Microsoft.Azure.Attestation/src/Generated/CertsOperations.cs

```diff
@@ -0,0 +1,249 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Attestation
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// CertsOperations operations.
+    /// </summary>
+    internal partial class CertsOperations : IServiceOperations<AttestationClient>, ICertsOperations
+    {
+        /// <summary>
+        /// Initializes a new instance of the CertsOperations class.
+        /// </summary>
+        /// <param name='client'>
+        /// Reference to the service client.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        internal CertsOperations(AttestationClient client)
+        {
+            if (client == null)
+            {
+                throw new System.ArgumentNullException("client");
```

Comment:
`nameof(client)`

Reply:
This is an automated generated file by running autorest ,  I can't modify it.